<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationsAndTags</class>
 <widget class="QWidget" name="AnnotationsAndTags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnnotationsAndTags</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelCategory">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditCategory"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelApplicationArea">
     <property name="text">
      <string>Application Area</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditApplicationArea"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDataVersion">
     <property name="text">
      <string>Data Version</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditDataVersion"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelLastModified">
     <property name="text">
      <string>Last Modified</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditLastModified"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
